src/style/style.scss

```diff
@@ -1,17 +1,21 @@
-@import 'globalvars';
-@import 'container';
-
-$textcolor: #000;
+@import "globals";
 
+body,
 html,
 body,
 button {
-  color: $textcolor;
-  font-family: 'Varela Round', sans-serif !important;
+  color: rgb(60, 66, 87);
+  font-family: -apple-system , BlinkMacSystemFont , "Segoe UI" , "Roboto" , "Helvetica Neue" , "Ubuntu" , sans-serif;
+  -webkit-font-smoothing: antialiased;
+  text-rendering: optimizeLegibility;
+}
+
+h1, h2, h3, nav {
+  font-family: 'Varela Round', sans-serif;
```

Comment:
This removes the 'Varela Round' font for a lot of header tags which should use it.

Reply:
Fixed in new PR